modules/client/src/addressList/internal/graphql-queries/proposal.ts

```diff
@@ -1,8 +1,8 @@
 import { gql } from "graphql-request";
 
-export const QueryAddressListProposal = gql`
-query addressListProposal($proposalId: ID!) {
-  addresslistProposal(id: $proposalId){
+export const QueryAddressListVotingProposal = gql`
+query addressListVotingProposal($proposalId: ID!) {
```

Comment:
This query has a lowercase first character on the name, whereas above, `AddressListVotingPluginMembers` has an uppercase one.

Does this come from Subgraph?

Reply:
The thing is that in subgraph and contracts we have `addresslistVoting`, and in the sdk we have a mix of `addressList` and `addressListVoting` I would do de change on the sdk to unify the naming but I'd do it in a separate PR that also addreses another naming changes such as `ClientAddressList` => `AddresslistClient`

Reply:
Added to APP-1514 👍